### PR TITLE
Test case for EDITNNCR

### PR DESCRIPTION
**Pull request by tskille**
This test case is very similar to the existing test case for EDITNNC. Only difference is that this is using EDITNNCR in stead of EDITNNC.

EDITNNCR is currently not supported, but hopefully this will come inn soon.

### CODE REVIEW

File: editnnc/EDITNNCR.DATA [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+ 2697.0   106.77 /
+
+
+------------------------------------------------------------------------------------------------
+SUMMARY
+------------------------------------------------------------------------------------------------
+
+DATE
+-- 
+RUNSUM
```
Comment (bska) [inline editnnc/EDITNNCR.DATA:227]: If the ultimate goal is to have this model in the regression test suite, then I'd prefer if we did not generate `.RSM` files.  We've experienced problems getting stable runs if there is a `.RSM` file in the result set.

**Review by bska — COMMENTED** [review]
Looks good except for one tiny nit concerning `RUNSUM`.  You might also wish to fix the misprint of "repeated" in the [introductory header](https://github.com/OPM/opm-tests/blob/002301876a94211f1529f464c369dd8852eb95f6/editnnc/EDITNNCR.DATA#L157) for `EDITNNCR`.

File: editnnc/EDITNNCR.DATA (hunk 1 above)
Comment (tskille) [reply editnnc/EDITNNCR.DATA]: The request for a .RSM file was not intentional.  I have also added one extra include file which replace and EDITNNC keyword with a EDITNNCR keyword.  The test case now actually includes repeated entries for some of the NNCs.  

Thank you for checking so carefully my test cases. Really appreciated.

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  I'll merge into master.

**Pull request merged**